### ISSUE TITLE
Timeline drag-and-drop does not work in Internet Explorer

### ISSUE THREAD

**Issue by gulench** [almende/vis#2370]
The drag-and-drop example, [examples/timeline/other/drag&drop.html](https://github.com/almende/vis/blob/v4.17.0/examples/timeline/other/drag%26drop.html), does not work in Internet Explorer (versions ..., 10, 11).

This is the error that gets logged: `SCRIPT438: Object doesn't support property or method 'includes'` in [lib/timeline/Core.js](https://github.com/almende/vis/blob/v4.17.0/lib/timeline/Core.js#L242):

```js
    function handleDragOver(event) {
      ...
      // make sure your target is a vis element
      if (!event.target.className.includes('vis')) return;
      ...
```

**Comment by yotamberk** [comment]
Works for IE11. See #2372

**Comment by mojoaxel** [comment]
@yotamberk What about IE10? Should we reopen this?

**Comment by yotamberk** [comment]
Fixes with #2426
